src/components/Sqlninja/sqlninja.tsx

```diff
@@ -0,0 +1,225 @@
+        setAllowSave(false);
+
+        // Construct the argmentss for the sqlninja command based on form input.
+        const args = [];
+        // Adds the argument for mode.
+        selectedMode ? args.push(`-m`, selectedMode) : undefined;
+        // Adds the file argument if a filepath is input.
+        if (values.filePath) {
+            args.push("-f", values.filePath);
+        }
+        // Adds verbose output argument if switch is on.
+        if (checkedVerbose) {
+            args.push(`-v`);
+        }
+
+        // Execute the bash command via helper method and handle its output or potential errors.
+        CommandHelper.runCommandGetPidAndOutput("sqlninja", args, handleProcessData, handleProcessTermination)
+            .then(({ pid, output }) => {
+                setPid(pid);
+                setOutput(output);
+            })
+            .catch((error) => {
+                setLoading(false);
+                setOutput(`Error: ${error.message}`);
+            });
+    };
+
+    /**
+     * clearOutput: Callback function to clear the console output.
+     * It resets the state variable holding the output, thereby clearing the display.
+     */
+    const clearOutput = useCallback(() => {
+        setOutput("");
+        setHasSaved(false);
+        setAllowSave(false);
+    }, [setOutput]);
+
+    return (
+        <RenderComponent
+            title={title}
+            description={description}
+            steps={steps}
+            tutorial={tutorial}
+            sourceLink={sourceLink}
+        >
+            {!loadingModal && (
+                <InstallationModal
+                    isOpen={opened}
+                    setOpened={setOpened}
+                    feature_description={description}
+                    dependencies={dependencies}
+                ></InstallationModal>
+            )}
+            <form onSubmit={form.onSubmit((values) => onSubmit(values))}>
+                {LoadingOverlayAndCancelButton(loading, pid)}
+                <Stack>
+                    <NativeSelect
+                        value={selectedMode}
+                        onChange={(e) => setSelectedMode(e.target.value)}
+                        title={"Attack Mode"}
+                        data={attackMode}
+                        placeholder={"Attack Mode"}
+                        description={"Please select attack mode"}
+                        required
+                    />
+                    <TextInput
+                        label={"Configuration File Path"}
+                        placeholder="default: sqlninja.conf"
+                        {...form.getInputProps("filePath")}
+                    />
+                    <Switch
+                        label="Verbose Mode"
+                        checked={checkedVerbose}
+                        onChange={(e) => setCheckedVerbose(e.currentTarget.checked)}
+                    />
+                    <Button type={"submit"}>Start</Button>
```

Comment:
You should use Start {title}
```suggestion
                    <Button type={"submit"}>Start {title}</Button>
```